sdk/storage/storage-file-datalake/test/pathclient.spec.ts

```diff
@@ -939,7 +940,7 @@ describe("DataLakePathClient", () => {
 
     await tempFileClient.flush(body.length * 2, {
       proposedLeaseId: leaseId,
-      leaseDuration: 15,
+      leaseDurationInSeconds: 15,
```

Comment:
should this change be added to change log?

Reply:
Updated